store/src/java/com/zimbra/cs/redolog/RedoLogManager.java

```diff
@@ -354,7 +373,10 @@ public synchronized void stop() {
         }
 
         try {
-            forceRollover();
+            // rollover only is needed when use FileLogWriter mechanism
+            if (getLogWriter() instanceof FileLogWriter) {
```

Comment:
Change to `!(mLogWriter instanceof DbLogWriter)`. Use negation to ensure that we are changing the behavior for only the new `DbLogWriter`.